domains/POAS/events/reduced-fee-requested/schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.makeregister/reduced-fee-requested.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.makeregister/reduced-fee-requested",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "M-([A-Z0-9]{4}){3}"
+    },
+    "requestType": {
+      "type": "string",
+      "description": "The type of reduction being requested",
+      "enum": ["HalfFee", "NoFee", "HardshipFee"]
```

Comment:
As of last week we had it [confirmed](https://opgtransform.atlassian.net/browse/MLPAB-1380) `HardshipFee` is not going to be required.